API Documentation plugin callout (#900)
strapi/documentation: docs/developer-docs/latest/plugins/documentation.md

# API Documentation

::: caution

> The Documentation plugin documentation may not be fully updated for Strapi v4. Improving the plugin and the documentation is an active project, which should be completed by the end of September 2022.

Reviewer: ```suggestion
The Documentation plugin documentation may not be fully updated for Strapi v4. The Strapi team is currently improving that plugin and documentation, which should be completed by the end of September 2022.
```
Or something like that. I thought it'll be nice to mention the Strapi team in the message to make it a bit more "personal". What do you think?
